node/subsystem-types/src/errors.rs

```diff
@@ -21,8 +21,8 @@ use crate::JaegerError;
 /// A description of an error causing the runtime API request to be unservable.
 #[derive(thiserror::Error, Debug, Clone)]
 pub enum RuntimeApiError {
-	/// The runtime API cannot be executed due to a
-	#[error("The runtime API '{runtime_api_name}' cannot be executed")]
+	/// The runtime API cannot be executed due to a runtime error.
+	#[error("The runtime API '{runtime_api_name}' cannot be executed: {source}")]
```

Comment:
nit: `source` will be printed twice now in the backtrace, once as part of the backtrace and as part of the error message itself iirc

Reply:
Do you know how you'd have to print it out to actually get it printed out twice?

Because when printed out through a normal `Display` trait:

```
    #[test]
    fn test_foobar() {
        #[derive(thiserror::Error, Debug)]
        #[error("source")]
        struct Source;
        let err = RuntimeApiError::Execution { runtime_api_name: "Test", source: Arc::new(Source) };
        panic!("{}", err);
    }
```

it's printed out like this:

```
thread 'import::tests::test_foobar' panicked at 'The runtime API 'Test' cannot be executed: source', node/core/approval-voting/src/import.rs:1351:9
```

And we usually print out errors through `Display` (or `Debug`).